creator-node/src/fileManager.js

```diff
@@ -135,6 +153,9 @@ async function saveFileForMultihashToFS (req, multihash, expectedStoragePath, ga
     // If file already stored on disk, return immediately.
     if (fs.existsSync(expectedStoragePath)) {
       req.logger.debug(`File already stored at ${expectedStoragePath} for ${multihash}`)
+      decisionTree.push({ stage: 'File already stored on disk', vals: [expectedStoragePath, multihash, Date.now()] })
```

Comment:
lolol, i assume you meant `vals: [expectedStoragePath, multihash], time: Date.now()`

Reply:
i did